themes/default/content/blog/pulumi-is-imperative-declarative-imperative/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Pulumi Is Imperative, Declarative, and Imperative"
+title: "Pulumi's Declarative and Imperative Approach IaC"
```

Comment:
> Pulumi's Declarative and Imperative Approach to IaC"

Missing a "to".

We can also add `allow_long_title: true` to get past the 60-character link check if we need to.